extensions/Worker.Extensions.Abstractions/src/ExtensionInformationAttribute.cs

```diff
@@ -14,24 +14,16 @@ public class ExtensionInformationAttribute : Attribute
 
         public bool EnableImplicitRegistration { get; }
 
-        public bool SupportsDeferredBinding { get; }
-
         public ExtensionInformationAttribute(string extensionPackage, string extensionVersion)
-            : this(extensionPackage, extensionVersion, false, false)
+            : this(extensionPackage, extensionVersion, false)
         {
         }
 
         public ExtensionInformationAttribute(string extensionPackage, string extensionVersion, bool enableImplicitRegistration)
-            : this(extensionPackage, extensionVersion, enableImplicitRegistration, false)
-        {
-        }
-
-        public ExtensionInformationAttribute(string extensionPackage, string extensionVersion, bool enableImplicitRegistration, bool supportsDeferredBinding)
```

Comment:
I think removing this constructor would be a breaking change.

Reply:
We should be okay; we added this constructor and nothing is using it yet other than the draft SDK-Binding PRs. Plan is to switch to using the new attribute instead of this constructor before we release anything and cause breaking changes later

Reply:
Oh, so this has not been released yet?

Reply:
Here's the last [PR](#1107) for reference, I don't believe we ever published this and if we ever did it was preview

Reply:
If it has not been released (or only released as preview), then I have no worries about the change.